libbeat/beat/beat.go

```diff
@@ -28,14 +28,13 @@ import (
 	"runtime"
 	"sync"
 
-	"github.com/satori/go.uuid"
-	"github.com/urso/ucfg"
-
 	"github.com/elastic/beats/libbeat/cfgfile"
+	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/filter"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/publisher"
 	"github.com/elastic/beats/libbeat/service"
+	"github.com/satori/go.uuid"
```

Comment:
We should probably write down what rule we are following here. So far I tried to do:

* go shipped packages
* external packages from any source
* internal packages

WDYT?